proxy/ParentSelection.cc

```diff
@@ -524,6 +557,7 @@ ParentRecord::Init(matcher_line *line_info)
   const char *tmp;
   char *label;
   char *val;
+  char parent_buf[8192]  = {0};
```

Comment:
Also, where did this size come from? Just arbitrary? Or can we correlate this to some reasonable max length?